public/assets/js/client.js

```diff
@@ -56,9 +55,13 @@ function postSignUpData(person) {
                 address: person.address,
                 zipcode: person.zipcode
             })
-        }).then((res) => res.json())
-        .then((data) => {
-            console.log("Success");
+    }).then(res => res.json())
+      .then((data) => {
+          console.log("Success --> ", data);
+          const userId = data.message;
+          const queryUrl = '/profile/' + userId;
+          console.log(queryUrl);
+          window.location.href = '/profile/' + userId;
```

Comment:
This is the redirect.  The route in `userController.js` requires an ID, so I built the url accordingly.  The `queryUrl` part is unnecessary, I'll remove it soon.